code/game/objects/structures/statues.dm

```diff
@@ -300,3 +300,13 @@
 	desc = "A bust depicting a certain 19th century economist. You get the feeling a specter is haunting the station."
 	icon_state = "marx"
 	art_type = /datum/component/art/rev
+
+///////////Elder Atmosian///////////////////////////////////////////
+
+/obj/structure/statue/elder_atmosian
+	name = "Elder Atmosian"
+	desc = "A statue of an Elder Atmosian, capable of bending the laws of thermodinamics at his/her will"
+	icon_state = "eng"
```

Comment:
😏

Reply:
Could we not just have "their will"? Sounds more natural. Otherwise, just pick the gender that the statue looks most like